framework/sessions/__init__.py

```diff
@@ -158,7 +158,7 @@ def before_request():
     cookie = request.cookies.get(settings.COOKIE_NAME)
     if cookie:
         try:
-            session_id = itsdangerous.Signer(settings.SECRET_KEY).unsign(cookie)
+            session_id = itsdangerous.Signer(settings.SECRET_KEY).unsign(cookie).decode()
```

Comment:
Nice. Now I remember why I went back to `develop` to check `fakecas`, which happened to be broken due to a different issue (local same-site has to be Lax using HTTP). It gave me the illusion that it fixed my upgraded django session as well. The truth is: my local upgraded OSF sign-in doesn't work until I applied your change.

```
web_1               | [framework.sessions]  WARNING: session_id is b'6318f9e063ba6500816b75f2'
web_1               | [website.project.decorators]  WARNING: user found in session is None
```

vs

```
web_1               | [framework.sessions]  WARNING: session_id is 6318f9e063ba6500816b75f2
web_1               | [framework.sessions]  WARNING: user found in session is ympkn
```